Formula/percona-server.rb

```diff
@@ -72,13 +72,8 @@ def install
       -DCOMPILATION_COMMENT=Homebrew
       -DWITH_EDITLINE=system
       -DCMAKE_BUILD_TYPE=RelWithDebInfo
-    ]
-
-    # PAM plugin is Linux-only at the moment
```

Comment:
Can you, in the PR (not in the formula) give a link that confirms PAM works for macOS now? (release notes or something like that)